pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -69,6 +70,14 @@ const (
 	// ResultTimeout is the time in seconds we wait for a result to return from the gadget
 	// after sending a Stop command
 	ResultTimeout = 30
+
+	// ConnectionMethodTCP uses the Kubernetes API Server using port-forwarding to connect to the
```

Comment:
nit: perhaps `ConnectionProtocol` is better?

Reply:
Hmm, I don't know - sure, "tcp" is a protocol, but is "unix" as well?

Reply:
Right, "method" is better then.